octokit/octokit_test.go

```diff
@@ -122,3 +122,24 @@ func loadFixture(f string) string {
 	c, _ := ioutil.ReadFile(p)
 	return string(c)
 }
+
+func stubGet(t *testing.T, path, fixture string, params map[string]string) {
+	stubRequest(t, "GET", path, fixture, params)
+}
+
+func stubRequest(t *testing.T, method string, path string, fixture string, params map[string]string) {
```

Comment:
Could prob. pass a func at the last for assertion or optional manipulation as well:

```go
func stubRequest(t *testing.T, method string, path string, fixture string, params map[string]string, f func(w http.ResponseWriter, r *http.Request) {
  ...

  mux.HandleFunc(path, func(w http.ResponseWriter, r *http.Request) {
       if f != nil {
          f(w, r)
       } 
  }
}
```